packages/types/src/interview.ts

```diff
@@ -22,9 +22,12 @@ export type Self = {
      * The interviewee id (the id of the tutor)
      */
     interviewee: number;
-    call: number;
     rule: number;
+    session: ISession.Id; /** @deprecated */
   };
+  // sessionId has to be extracted from ids to be compatible with ILesson.Self 
+  // as ISession.Self is either a lesson.self or interview.self
+  sessionId: ISession.Id;
```

Comment:
We can remove this. 


```suggestion
```

Reply:
In case you didn't notice... It's value showcases (here)[apps/nova/src/components/TutorOnboardingSteps/Interview/ScheduledInterview/PendingInterview/index.tsx]; removing it will just make the code more tedious.

Reply:
The frontend code that is using it will be removed. It is find to remove `sessionId` and comment out the frontend code.

Reply:
Got it.